app/models/hold_source.rb

```diff
@@ -0,0 +1,12 @@
+# == Schema Information
+#
+# Table name: hold_sources
+#
+#  id         :integer          not null, primary key
+#  source     :text             not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
```

Comment:
Similarly to the Holds mode, please add validations to this model to allow Rails to know when an Object is valid and not. You can see why quickly in this model in the `rails console` by:
hs = HoldSource.new
hs.valid?

That will be true. It is not :)